### PR TITLE
Removed CrossIndex Class

### PR DESCRIPTION
**Pull request by p365labs**
Removed deprecated CrossIndex, use Reindex. Issue #1404

### CODE REVIEW

**Comment by ruflin** [comment]
@p365labs Could you add a CHANGELOG entry?

**Comment by p365labs** [comment]
yep sorry

**Comment by p365labs** [comment]
@ruflin done :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file based on the
 
 ### Backward Compatibility Breaks
 - Return the [_source of inner hit nested](https://github.com/elastic/elasticsearch/pull/26982) as is without wrapping it into its full path context [#1398](https://github.com/ruflin/Elastica/pull/1398)
+- Removed CrossIndex Class as from now use Reindex.
```
Comment (ruflin) [inline CHANGELOG.md]: Sorry to be picky, but could you add the PR number and a link to the ES reindex docs?

**Comment by p365labs** [comment]
no problem done

**Pull request merged**